packages/core/src/layout/FlexLayout/FlexLayout.tsx

```diff
@@ -93,7 +93,11 @@ export const FlexLayout = forwardRef<HTMLDivElement, FlexLayoutProps>(
         style={flexLayoutStyles}
         {...rest}
       >
-        {children}
+        {separators
+          ? Children.map(children, (child) => (
```

Comment:
does feel a shame to have to introduce this extra layer just when we have separators. Don't have a better suggestion though

Reply:
I guess the other option would be creating an explicit separator component, which will probably the what's needed once we start thinking about the splitter, for flexlayout implementation wise we will still need to go trough each child and append it, but that would get rid of the wrapper